package/opencdm/opencdm.mk

```diff
@@ -8,8 +8,8 @@ OPENCDM_SITE_METHOD = git
 OPENCDM_SITE = https://github.com/WebPlatformForEmbedded/WPEOpenCDM.git
 
 OPENCDM_INSTALL_STAGING = YES
-OPENCDMI_LICENSE = Apache-2.0
-OPENCDMI_LICENSE_FILES = LICENSE
+OPENCDM_LICENSE = Apache-2.0
```

Comment:
Perhaps these unrelated changes should be in a separate patch.

Reply:
Ok will remove it from this